content/blog/2018/07/2018-07-12-simple-pull-request-plugin.adoc

```diff
@@ -0,0 +1,193 @@
+reports:
+  - location_of_report_1
+  - location_of_report_2
+
+environment:
+  variables:
+    variable_1: value_1
+    variable_2: value_2
+
+  # Credentials contains only two fields. credentialId must be present in the Jenkins Credentials
+  credentials:
+    - credentialId : fileCredentialId
+      variable : FILE
+
+      # In user scripts Username and Password can be accessed by LOGIN_USR and LOGIN_PSW
+      # respectively as environment variales
+    - credentialId : dummyGitRepo
+      variable : LOGIN
+
+stages:
+  - name: stage1
+    agent: any
+    steps:
+      - sh: "scripts/hello"
+      - sleep:
+          time: 2
+          unit: SECONDS
+      - sleep: 2
+      - junit:
+          testResults: "target/**.xml"
+          allowEmptyResults: true
+          testDataPublishers:
+            - AutomateTestDataPublisher
+            - JunitResultPublisher:
+                urlOverride: "urlOverride"
+    # Post section for "stage1". All Conditions which are available in Jenkins
+    # declarative pipeline are supported
+    post:
+      failure:
+        - sh: "scripts/hello"
+
+# Outer post section. Just like declarative pipeline.
+post:
+  always:
+    - sh: "scripts/hello"
+----
+
+=== Coding Phase 2 plans
+
+* [*] Decide a proper YAML format to use for Jenkinsfile.yaml
+* [*] Create Step Configurator for SPRP plugin. https://issues.jenkins-ci.org/browse/JENKINS-51637[JENKINS-51637].
+This will enable users to use Pipeline steps in Jenkinsfile.yaml.
+* [*] Automatic indentation generation in the generated PipelineSnippetGenerator class.
+* [*] Write tests for the plugin.
+
+=== Coding Phase 3 plans
+
+. Support for webhooks.
+. Mechanism to check if pull request is approved by trusted users.
+. Support report types.
+. Conditions like when.
+. Prepare documentation.
+
+
+https://issues.jenkins-ci.org/browse/JENKINS-52452[Phase 3 Jira Epic]
+
+
+=== How to reach me
+
+* Email: gautamabhishek46@gmail.com
+* Gitter room: link:https://gitter.im/Jenkinsci/simple-pull-request-job-plugin[]
+
+=== References
+
+* link:https://docs.google.com/document/d/1cuC0AvQG3e4GCjIoCwK3J0tcJVAz1eNDKV8d_zXxlO8/edit[Initial proposal of the project]
+* link:https://github.com/jenkinsci/simple-pull-request-job-plugin[Project repository]
+* link:https://jenkins.io/projects/gsoc/2018/simple-pull-request-job-plugin/[Project page]
+* link:https://gitter.im/jenkinsci/simple-pull-request-job-plugin?utm_source=share-link&utm_medium=link&utm_campaign=share-link[Gitter chat]
+* link:https://issues.jenkins-ci.org/issues/?jql=project%20%3D%20Jenkins%20AND%20component%20%3D%20simple-pull-request-job-plugin[Bug Tracker]
+* link:https://github.com/gautamabhishek46/dummy[Demo Repository]
+* link:https://www.youtube.com/watch?v=qWHM8S0fzUw[Phase 2 Presentation video](July 12, 2018)
+* link:https://docs.google.com/presentation/d/1V3hHEHHkJlwI3sK1VTIDOvbflXMtdn_JUc9cOFIrOOQ/edit?usp=sharing[Phase 2 Presentation Slides](July 12, 2018)
```

Comment:
Technically you could just embed slides and video into your blogpost somewhere. Example: https://raw.githubusercontent.com/jenkins-infra/jenkins.io/master/content/blog/2018/03/2018-03-15-jep-200-lts.adoc